### ISSUE TITLE
[React Native New Architecture]: User is not able to navigate to any static texts using 'caps lock +arrow keys' or 'arrow keys in scan mode'.

### ISSUE THREAD

**Issue by HariniMalothu17** [microsoft/react-native-gallery#589]
Build Details:
OS Version: Dev (OS Build 27881.1000)
App: React Native Gallery Experimental
AT: Narrator

 

Repro-Steps:
Launch the React Native Gallery- Experimental app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to 'Home' button and invoke it.
Navigate through the page using 'caps lock +arrow keys' and 'arrow keys in scan mode'.
Observe the Narrator behavior.
Actual Result:
User is not able to navigate through the static texts using 'caps lock +arrow keys' or 'arrow keys in scan mode'. On hitting caps arrow keys or only arrow keys in scan mode, Narrator announces as "Unable to move Narrator cursor".
Note: Same issue is observed throughout the application.

Expected Result:
User should be able to navigate through all the static texts using 'caps lock +arrow keys' or 'arrow keys in scan mode'. 

User impact:

Users with visual impairment who rely on screen reader will face difficulties if they won't be able to navigate through all the static texts.

MAS Reference:  [MAS 4.2.1 – Object Information](https://aka.ms/MAS4.2.1)
 
Trap ID: [1.4 - Uncomprehended Element](https://aka.ms/UncomprehendedElement)
 
“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by vineethkuttan** [comment]
The Navigation part for caps+arrow is working, since Text components couldn't  able get the Narrator focus. It looks like CAPS + Arrow doesn't work.
But the scan mode is actually working though.

I guess fixing the text provider , will automatically fix this issue.
